src/components/common/transmission/transmission.ts

```diff
@@ -12,6 +12,9 @@ export class TransmissionComponent implements OnChanges {
   @Input()
   transmission: TransmissionType;
 
+  @Input()
+  hideLabel: boolean = false;
```

Comment:
Don't like the naming of this. Not sure it gives a clear intent.  it's hiding one label but showing another.
Just personal preference though.

Reply:
Yes, I totally agree with this looking back on it. I've pushed changes if you want to have another look.